panos/userid.py

```diff
@@ -246,6 +247,8 @@ def register(self, ip, tags):
             tagelement = register.find("./entry[@ip='%s']/tag" % c_ip)
             if tagelement is None:
                 entry = ET.SubElement(register, "entry", {"ip": c_ip})
+                if timeout:
```

Comment:
This if statement needs changed to move the timeout value into the member sub element not the entry sub element

Reply:
Please take a look and let me know if I got it right this time!